Enable SA1504: All accessors should be single-line or multi-line

xtqqczze opened the pull request:
https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1504.md

xtqqczze commented:
Depends on #14243.

iSazonov commented on src/Microsoft.Management.Infrastructure.CimCmdlets/GetCimAssociatedInstanceCommand.cs:
```diff
@@ -90,7 +90,10 @@ public string ResultClassName
         public CimInstance InputObject
         {
-            get { return cimInstance; }
+            get
+            {
+                return cimInstance;
+            }
```
It is auto fix? Is it possible to use modern `get => cimInstance;`?

xtqqczze replied:
There is an open issue for expression-bodied accessors: https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/2402

xtqqczze replied:
`get => cimInstance;` is not currently a violation, but may not be in the spirit of the rule.

xtqqczze committed:
Fix SA1504: All accessors should be single-line or multi-lin

https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1504.md
